src/schema/v2/sale/index.ts

```diff
@@ -131,6 +131,13 @@ export const SaleType = new GraphQLObjectType<any, ResolverContext>({
           return null
         },
       },
+      bidder: {
+        type: Bidder.type,
+        resolve: async ({ id }, _options, { meBiddersLoader }) => {
+          const bidders = await meBiddersLoader({ sale_id: id })
+          return first(bidders)
+        },
+      },
```

Comment:
fixme: this is the slowest part of the request cycle

Reply:
Yep, that sounds like a good idea. Batching from MP's side isn't that bad, but we'd have to enhance the gravity API first to support batching for that endpoint: https://github.com/artsy/gravity/blob/master/app/api/v1/me_bidders_endpoint.rb

Reply:
Added a note 👍